pyproject.toml

```diff
@@ -0,0 +1,84 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[tool.hatch.version]
+source = "vcs"
+
+[tool.hatch.build.targets.sdist]
+include = [
```

Comment:
exclude `/.github` as well to not build with GitHub settings.

Reply:
This is already only including the wrapanapi subdir. From inspecting an sdist archive built through hatch, only `.gitignore`, `README.rst`, and `pyproject.toml` are retained from the top level directory.